### DISCUSSION TITLE
Welcome to LookinLoader Discussions!

### DISCUSSION THREAD

**Discussion by creantan** [creantan/LookinLoader#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tweaker177** [comment]
Hi.  Only really commenting because I never tried the "new" discussions feature and am about to create a lot of new repos. Seeing how I like it.  I mainly focus on tweak development for jailbroken iOS devices. For the longest time I never open sourced my best work, since the community was so full of people quick to copy and release inferior or partial versions of tweaks I spent hundreds of hours researching and developing.  

However now I've been doing a lot more with Python, Node.js, a little React Native, and basically anything helpful to full stack web3 programming.  Solidity too! Can’t forget that one lol.  Anyways, with my focus shifting, and having given up making any real money from tweak development, I'm going to open source a lot more jailbreak related projects, self-made tools, tweaks never released besides personal use (usually ones uncomfortably in gray area of hacking, not illegal or even bad in it's actual functionality, but usually there are use cases that the community at large may frown on, like getting Apple News+ for free without hacking the IAP nor App Store or even tricking it into thinking IAP was bought.  I just removed all paywalls.) 

Anyways, I do a LOT of research using variations of the Flex view explorer, runtime object modifier and inspector.  

Is this tweak fully working on iOS 15, and would it completely get rid of the need for a Flex debugging tool?  (From the screenshots it seems a lot more powerful and informative.)
